docs/g_slide_002.md

```diff
@@ -27,4 +27,4 @@ An example of "sensorimotor Lenia" pattern interacting with immutable obstacles
 [^Ha2022]: Hamon _et al._ 2022 [https://developmentalsystems.org/sensorimotor-lenia](https://developmentalsystems.org/sensorimotor-lenia)
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/g_slide_001) -- [Next slide](https://rivesunder.github.io/yuca/g_slide_003)
+[Previous slide](https://rivesunder.github.io/yuca_docs/g_slide_001) -- [Next slide](https://rivesunder.github.io/yuca_docs/g_slide_003)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.